### PR TITLE
search 7.x | les learning to rank port | LRDOCS-7777

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-7777
https://issues.liferay.com/browse/LRDOCS-8505

### CODE REVIEW

File: docs/dxp/7.x/en/using-search/liferay_enterprise_search.rst [hunk 1]
```diff
@@ -7,6 +7,37 @@ Liferay Enterprise Search
    liferay-enterprise-search/cross_cluster_replication.rst
    liferay-enterprise-search/learning-to-rank.md
 
+A Liferay Enterprise Search (LES) subscription gets you additional features beyond what's available out of the box with your Liferay DXP subscription. It includes
```
Comment (lipusz) [inline docs/dxp/7.x/en/using-search/liferay_enterprise_search.rst:10]: This intro/overview should be reworded. For example, it's not only Monitoring that you get a license for. X-pack is free, but still need LES on ES 6.x..

We should also probably list the available components or link to https://help.liferay.com/hc/en-us/articles/360014400932#Components.


The "high level steps" are not needed.

Comment (lipusz) [reply docs/dxp/7.x/en/using-search/liferay_enterprise_search.rst:10]: @rbohl :point_up:

Comment (rbohl) [reply docs/dxp/7.x/en/using-search/liferay_enterprise_search.rst:10]: @lipusz I just want to confirm that after making this change, both LTR and the LES intro are SME Reviewed, and I can send it in for publication?

Comment (lipusz) [reply docs/dxp/7.x/en/using-search/liferay_enterprise_search.rst:10]: I'd like to review the overview after the changes if possible before forwarding.

Comment (rbohl) [reply docs/dxp/7.x/en/using-search/liferay_enterprise_search.rst:10]: Yep, no problem

**Comment by rbohl** [comment]
Newly pushed commit--probably not at the final stage of refinement, but I wanted your feedback. 
les-intro

**Comment by lipusz** [comment]
:+1: LGTM. :ship:

**Comment by lipusz** [comment]
Could we forward the monitoring PR too to JR? I really would like these pages to be published asap.

**Comment by lipusz** [comment]
Thank you, Russell!

**Comment by rbohl** [comment]
Superseded by https://github.com/sez11a/liferay-learn/pull/378

Note that I added one more commit--the content still seemed choppy to me, so I tried to make it read more smoothly. Let me know if I've messed anything up.

**Pull request closed without merging**